shell/platform/tizen/public/flutter_tizen.h

```diff
@@ -209,12 +209,17 @@ FLUTTER_EXPORT void FlutterDesktopViewOnPointerEvent(
     size_t timestamp,
     int32_t device_id);
 
-FLUTTER_EXPORT void FlutterDesktopViewOnKeyEvent(FlutterDesktopViewRef view,
-                                                 const char* key,
-                                                 const char* string,
-                                                 uint32_t modifiers,
-                                                 uint32_t scan_code,
-                                                 bool is_down);
+FLUTTER_EXPORT void FlutterDesktopViewOnKeyEvent(
+    FlutterDesktopViewRef view,
+    const char* key,
+    const char* string,
+    uint32_t modifiers,
+    uint32_t scan_code,
+    bool is_down,
+    size_t timestamp = 0,
+    const char* device_name = nullptr,
+    uint32_t device_class = 0,
+    uint32_t device_subclass = 0);
```

Comment:
Does the C standard support "default arguments" or function overloading?

Reply:
Default argument is c++ feature. 
But because of `extern "C"`, symbols will be saved without `default argument` applied.(Maybe?) and since there are no duplicate symbols, it doesn't seem to be a problem now.
Is there any good way?
(The failure of githubaction doesn't seem to be due to this?)

Reply:
[Change to default parameter](https://github.com/flutter-tizen/engine/pull/349/commits/a98e9981a23551c9ff656e9d10418994d8456d34)
It is good to revert this patch and re-consider the API for NUI when defining the struct type.